packages/hint/src/lib/hint-context.ts

```diff
@@ -125,7 +125,7 @@ export class HintContext<E extends Events = Events> {
 
     /** Reports a problem with the resource. */
     public report(resource: string, message: string, options: ReportOptions = {}) {
-        const { codeSnippet, element, severity } = options;
+        const { codeSnippet, element, severity = Severity.error } = options;
```

Comment:
Given our discussions thus far, I think we should use `Severity.warning` here since it will be the more common value.
```suggestion
        const { codeSnippet, element, severity = Severity.warning } = options;
```

Reply:
SGTM